modules/gitops-release-notes-1-12-0.adoc

```diff
@@ -0,0 +1,97 @@
+:_mod-docs-content-type: REFERENCE
+
+[id="gitops-release-notes-1-12-0_{context}"]
+= Release notes for {gitops-title} 1.12.0
+
+{gitops-title} 1.12.0 is now available on {OCP} 4.12, 4.13, and 4.14.
+
+[id="errata-updates-1-12.0_{context}"]
+== Errata updates
+
+TBD
+
+[id="new-features-1-12-0_{context}"]
+== New features
+
+* With this update, the Argo CD CLI is supported and accessible as a productized and easily-installable RPM. The Argo CD CLI tool enables users to manage Red Hat OpenShift GitOps from a terminal. Using this tool, users can manage ArgoCD resources (Applications, ApplicationSets, AppProjects, User accounts, GPG keys etc) from a client terminal. link:https://issues.redhat.com/browse/GITOPS-3389[GITOPS-3389]
+
+[NOTE]
+====
+The `argocd` executable binary file is included in the archives and the RPMs.
+====
+
+[WARNING]
+==== 	
+The Argo CD CLI tool is a Technology Preview feature only.
+====
+
+* With this update, users can configuration notifications in a dedicated `NotificationsConfiguration` custom resource (CR). This process replaces the workflow in earlier releases of {gitops-title}. To configure this feature in earlier releases, you had to edit the `argocd-notifications-cm` ConfigMap manually. Users can now set Triggers, Templates, Services, and Subscriptions as fields in the new `NotificationsConfiguration` custom resource (CR), which is managed by the {gitops-title} Operator. link:https://issues.redhat.com/browse/GITOPS-4130[GITOPS-4130]
+
+* With this update, two new fields `.spec.applicationSet.sourceNamespaces` and `.spec.applicationSet.scmProviders` are introduced in ArgoCD CRD to support `ApplicationSet` in any namespaces. Administrators use it to define certain namespaces that manage `ApplicationSet` resources. link:https://issues.redhat.com/browse/GITOPS-3754[GITOPS-3754]
+
+* With this update, Argo CD server has the required permissions to manage `ApplicationSet` resources. link:https://issues.redhat.com/browse/GITOPS-3762[GITOPS-3762]
+
+* With this update, you can configure log levels such as `debug`, `info`, `warn`, `error`, `panic` and `fatal`. The default log level set for the output is `info`. Users can select an appropriate log level by adding the environment variable `LOG_LEVEL` in Red Hat OpenShift GitOps subscription's field, `.spec.config.env`. link:https://issues.redhat.com/browse/GITOPS-4016[GITOPS-4016]
+
+.Example
+[source,yaml]
+----
+   yaml
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: gitops-operator
+  namespace: openshift-gitops-operator
+spec:
+  ...
+  config:
+    env:
+    - name: LOG_LEVEL
+      value: "error"
+----
+
+* With this feature, Argo CD uses the wildcard values in `sourceNamespaces` to specify multiple namespaces or patterns of namespaces. To use this feature, specify the namespaces where Argo CD can manage applications in the Argo CD YAML with `.spec.sourceNamespaces`. link:https://issues.redhat.com/browse/GITOPS-3935[GITOPS-3935]
```

Comment:
```suggestion
* With this update, Argo CD accepts the wildcard values in the `sourceNamespaces` field so that you can specify multiple namespaces or patterns of namespaces. To use this feature, specify the namespaces where Argo CD can manage applications in the `.spec.sourceNamespaces` field of the `ArgoCD` CR. link:https://issues.redhat.com/browse/GITOPS-3935[GITOPS-3935]
```

Reply:
Fixed.